maintenance/wikia/cronjobs/AutoFollow.php

```diff
@@ -2,24 +2,43 @@
 
 require_once( dirname(__FILE__) . '/../../commandLine.inc' );
 
-$alreadyWatchedKey = 'autowatched-already';
+global $wgCityId;
+$contentLanguage = WikiFactory::getVarValueByName( 'wgLanguageCode', $wgCityId );
```

Comment:
As you're running the script in a wiki context you can simply get content language via ``$wgLanguageCode`` global